controllers/numaresourcesoperator_controller_test.go

```diff
@@ -133,7 +133,7 @@ var _ = Describe("Test NUMAResourcesOperator Reconcile", func() {
 		})
 	})
 
-	Context("without available machine config pools", func() {
+	Context("[openshift] without available machine config pools", func() {
```

Comment:
we should use labels also in the controller tests but too big of a deal yet

Reply:
addressed in last commit, thanks!